### ISSUE TITLE
Previous nicotine dependence

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6529]
**Preferred term label:** Previous nicotine dependence

**Synonyms**
Previous nicotine addiction 

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Nicotine dependence (to be created)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
Yeah, I do think it is better not to add "previous" and "current" as new terms -- we would have to do that for all of the behavioural abnormalities. Could you please look at the Clinical Course -- https://hpo.jax.org/app/browse/term/HP:0031797
Perhaps we should add a term for previous or something similar there?
